build/template-OneBranch-CI-libsandsamples.yaml

```diff
@@ -30,4 +30,24 @@ steps:
     solution: ${{ parameters.Solution }}
     msbuildArgs: '/p:RunCodeAnalysis=false /p:MsalClientSemVer=${{ parameters.MsalClientSemVer }} /p:SourceLinkCreate=true /p:ContinousIntegrationBuild=true'
     platform: ${{ parameters.BuildPlatform }}
-    configuration: ${{ parameters.BuildConfiguration }}
\ No newline at end of file
+    configuration: ${{ parameters.BuildConfiguration }}
```

Comment:
should we also enable tests in OneBranch pr, release and net only release builds?

Reply:
The PR build will use the same file. However, adding tests to net only release is a good idea. I can add in a separate PR.